styles.css

```diff
@@ -33,7 +33,23 @@ img{
 }
 
 @media(max-width:600px){
-  .main{
-    background:red;
+  body {
+    flex-direction: column;
   }
+  header {
+    order: 1;
+  }
+  .main {
+    order: 2;
+  }
+  aside {
+    order: 3;
+  }
+  footer {
+    order: 4;
+  }
+}
+
+header > h1, aside > h2, .main > h2, .main > div, footer > h3, footer > p {
```

Comment:
While this technically works, a shorthand for styling the direct child of something is `>`. So if we wanted to modify all the direct children of a `<header>`, we would write this...

```css
header > * {
  /* style goes here */
}
```
> The `>` means that we are targeting a direct child. The `*` is a wildcard that means we are targeting everything. Together, this all means that we are targeting **all** of the **direct children** of header.